app/services/invoices/refresh_draft_service.rb

```diff
@@ -69,6 +69,7 @@ def call
         calculate_result.raise_if_error!
 
         flag_lifetime_usage_for_refresh
```

Comment:
I think we can add the `flag_lifetime_usage_for_refresh` to the if body as well right?